### PR TITLE
Return 401 error when logged out

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose
Currently, if you are not logged in the WFL API returns a redirect response. However, swagger will return `{"status":"ok"}` in this case, presumably because the redirect was successful. This is confusing behavior because it makes it seem like the request itself was successful.

### Changes
Make the `authorize` function return a 401 error if a user is not logged in.

### Review Instructions
- No instructions.

### CODE REVIEW

File: src/zero/api/handlers.clj [hunk 1]
```diff
@@ -55,7 +55,7 @@
         (if-let [jwt (some-> request :oauth2/access-tokens :google :id-token
                              decode-jwt valid?)]
           (handler (assoc request :jwt jwt))
-          (response/redirect landing-uri))))))
+          (response/status (response/response "") 401))))))
```
Comment (tbl3rd) [inline src/zero/api/handlers.clj]: Seems a shame to waste that `""`.

Comment (tbl3rd) [reply src/zero/api/handlers.clj]: And maybe add an `Authenticate` header?

Comment (tbl3rd) [reply src/zero/api/handlers.clj]: https://tools.ietf.org/html/rfc7235#section-4.1
… though maybe Ring does that for us?

Comment (samanehsan) [reply src/zero/api/handlers.clj]: I'm not sure...I'll take a look!

Comment (samanehsan) [reply src/zero/api/handlers.clj]: It's not added by Ring, so I added it in :octocat: Lmk what you think!

**Pull request merged**